crAPI/services/web/nginx.conf.template

```diff
@@ -53,6 +53,20 @@ server {
     sub_filter_once off;
   }
 
+  location /email/ {
```

Comment:
@ricekot does non-production (local development) get affected by this change?

or no impact to non fat container, regular deployment? I assume in a regular deployment both /mail/ and directly accessing mailhog will work ok?

Reply:
/mail/ will always work when nginx is used and so will mailhog UI if its port is exposed.

Reply:
@ricekot Those are the core API paths. So we will add sub paths to them. When you say "avoid adding any pages", are you referring to reactJS pages with that route?

Reply:
Yeah, we don't want reactJS page to clash with a route that redirects to another service

Reply:
@ricekot The way I understand this, it should not matter, even if we have a ReactJS page named /identity/. AFAIK, the NGINX routes come into effect when a REACT page makes a XHR call to the REST APIs.

So when a /identity/ ReactJS page makes a XHR call to /identity/API, these are two different things.

Is my understanding incorrect?

Reply:
Hmm, not sure, will have to read about this. In the case of /email/ though - let's say we create a new reactJS page at /email/ - then what will I see when I enter http://localhost/email/ in my browser - mailhog or the new page?

Reply:
@mgiamberardino what is the behavior in such case above?

will ReactJS trap that /email/ or will the browser make a http call to localhost:80/email/

?